src/model/models.ts

```diff
@@ -0,0 +1,33 @@
+export enum WordType {
+  TAB = ' ',
+  NEWLINE = '\n',
+}
```

Comment:
?
I don't exactly know what that's supposed to mean.

Reply:
It is to format the code. Was at the moment the best solution we could find out.